src/patient-registration/patient-registration.component.tsx

```diff
@@ -128,11 +132,29 @@ export const PatientRegistration: React.FC = () => {
     };
 
     if (values.contactPersonGivenName) {
-      const contactPersonAttributes = {
-        attributeType: uuidContactPersonGivenName,
-        value: values.contactPersonGivenName,
-      };
-      patient.person.attributes.push(contactPersonAttributes);
+      const contactPersonAttributes = [
```

Comment:
We should ideally refactor this before we merge this into `develop` and `master`.